doc/man1/flux-pstree.rst

```diff
@@ -0,0 +1,231 @@
+==============
```

Comment:
Is this manpage worthy of being added to flux help output?  I would think so?

Reply:
I wasn't sure at this point if it rose to that level?

Reply:
Its certainly a "user tool", so not in the same class as lets say `flux keygen`.  But it's clearly a more advanced user tool.  I dunno, I would have leaned yes, but its definitely borderline.

Reply:
I went ahead and added it. Thanks for the suggestion!